integration-tests/__tests__/global_setup.test.js

```diff
@@ -13,14 +13,14 @@ const path = require('path');
 const runJest = require('../runJest');
 const {cleanup} = require('../utils');
 
-const DIR = path.join(os.tmpdir(), 'jest_global_setup');
+const DIR = path.join(os.tmpdir(), 'jest_global-setup');
```

Comment:
In multiple places this need to be changed to: `jest-global-setup` (and `jest-global-teardown` accordingly too)

Reply:
Right you are! Searched and fixed these files:
```
→ git status -s                                                                                                            
 M integration-tests/__tests__/global_setup.test.js
 M integration-tests/__tests__/global_teardown.test.js
 M integration-tests/global-setup/__tests__/setup1.test.js
 M integration-tests/global-setup/__tests__/setup2.test.js
 M integration-tests/global-setup/__tests__/setup3.test.js
 M integration-tests/global-setup/setup.js
 M integration-tests/global-teardown/__tests__/teardown1.test.js
 M integration-tests/global-teardown/__tests__/teardown2.test.js
 M integration-tests/global-teardown/__tests__/teardown3.test.js
 M integration-tests/global-teardown/teardown.js
```